commons/protocol/src/main/scala/sbt/protocol/Keys.scala

```diff
@@ -82,11 +82,29 @@ object KeyList {
   implicit val pickler: Pickler[KeyList] = genPickler[KeyList]
 }
 
+// TODO - This is a bad name...
```

Comment:
btw I saw https://github.com/naholyr/github-todos on twitter the other day and thought of you :-)